draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2543,11 +2547,11 @@ http://example.com/hls/35cdc008/audio/index.m3u8\n”,
 
      	<section anchor="urls-spec" title="URLs Spec">
 	<t>
-		The URLs spec type allows the uCDN to manage uCDN content or metadata objects
+		The "urls" spec type allows the uCDN to manage uCDN content or metadata objects
 		held by the dCDN based on the objects' URLs.
```

Comment:
Hmm...
Title: URLs Spec
This Paragraph: "urls" spec
Next Paragraph URLs spec

I think you may have changed this instance back when you didn't need to.

Reply:
There are two conventions for trigger spec types in use in the document
One is using the full spec name that is in the name of the chapter, another is using the actual JSON attribute name, for example URI regex match spec type and "uri-regex-match" spec type.